test/test_distributions.py

```diff
@@ -187,6 +187,10 @@ def sample(self, key, sample_shape=()):
     T(dist.VonMises, 2., 10.),
     T(dist.VonMises, 2., jnp.array([150., 10.])),
     T(dist.VonMises, jnp.array([1 / 3 * jnp.pi, -1.]), jnp.array([20., 30.])),
+    T(dist.ProjectedNormal, jnp.array([0., 0.])),
+    T(dist.ProjectedNormal, jnp.array([2., 3.])),
+    T(dist.ProjectedNormal, jnp.array([0., 0., 0.])),
+    T(dist.ProjectedNormal, jnp.array([-1., 2., 3.])),
```

Comment:
It will be better if we convert a test for (2,) shape and a test for (3,) shape to (1, 2) and (1, 3) shapes.